### PR TITLE
DM-10765: Replace existing WCS classes with SkyWcs

### PR DESCRIPTION
**Pull request by r-owen**

### CODE REVIEW

File: include/lsst/meas/modelfit/detail/polynomials.h [hunk 1]
```diff
@@ -34,8 +34,8 @@ namespace lsst { namespace meas { namespace modelfit { namespace detail {
  * vectors evaluates the polynomial (or computes a derivative).
 */
 template <int N>
-struct Vandermonde {
```
Comment (rearmstr) [inline include/lsst/meas/modelfit/detail/polynomials.h]: This doesn't seem to belong to this commit.

Comment (r-owen) [reply include/lsst/meas/modelfit/detail/polynomials.h]: Good point. I made it a separate commit with a suitable commit message
